### PR TITLE
Additional swift support, renaming classes for swift

### PR DESCRIPTION
**Pull request by milos1290**
What              | Where/Who
------------------|----------------------------------------
JIRA Issue        | [LP-11127](https://leanplum.atlassian.net/browse/LP-11127)
People Involved   | @milos1290

### CODE REVIEW

File: Leanplum-SDK/Classes/LPInbox.h [hunk 1]
```diff
@@ -28,6 +28,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 #pragma mark - LPInboxMessage interface
 
+NS_SWIFT_NAME(InboxMessage)
```
Comment (e7mac) [inline Leanplum-SDK/Classes/LPInbox.h:31]: If the Inbox is LeanplumInbox, I dont like InboxMessage not having a prefix.

Comment (milos1290) [reply Leanplum-SDK/Classes/LPInbox.h:31]: I'm thinking to rename it to LeanplumInbox and LeanplumInbox.Message ?

File: Leanplum-SDK/Classes/LPActionArg.h [hunk 2]
```diff
@@ -27,19 +27,28 @@
...
++ (LPActionArg *)argNamed:(NSString *)name withFile:(nullable NSString *)defaultValue
+NS_SWIFT_NAME(init(name:file:));
++ (LPActionArg *)argNamed:(NSString *)name withDict:(NSDictionary *)defaultValue
+NS_SWIFT_NAME(init(name:dictionary:));
++ (LPActionArg *)argNamed:(NSString *)name withArray:(NSArray *)defaultValue
+NS_SWIFT_NAME(init(name:array:));
++ (LPActionArg *)argNamed:(NSString *)name withAction:(nullable NSString *)defaultValue
+NS_SWIFT_NAME(init(name:action:));
++ (LPActionArg *)argNamed:(NSString *)name withColor:(UIColor *)defaultValue
+NS_SWIFT_NAME(init(name:color:));
```
Comment (e7mac) [inline Leanplum-SDK/Classes/LPActionArg.h:51]: Swift doesn't need `alloc` before `init`right?

File: Leanplum-SDK/Classes/MessageTemplates/ViewControllers/LPInterstitialViewController.m [hunk 3]
```diff
@@ -15,7 +15,7 @@ @interface LPInterstitialViewController ()
 
 @implementation LPInterstitialViewController
 
-+(LPInterstitialViewController * _Nullable)instantiateFromStoryboard
++(LPInterstitialViewController *)instantiateFromStoryboard
```
Comment (e7mac) [inline Leanplum-SDK/Classes/MessageTemplates/ViewControllers/LPInterstitialViewController.m:18]: this is non null ?

File: Leanplum-SDK/Classes/LPActionArg.h (hunk 2 above)
Comment (milos1290) [reply Leanplum-SDK/Classes/LPActionArg.h:51]: no, this is just to tell that this method will be converted to constructor

File: Leanplum-SDK/Classes/MessageTemplates/ViewControllers/LPInterstitialViewController.m (hunk 3 above)
Comment (milos1290) [reply Leanplum-SDK/Classes/MessageTemplates/ViewControllers/LPInterstitialViewController.m:18]: hmm should be nullable

**Pull request merged**